rust/arguments.rs

```diff
@@ -0,0 +1,11 @@
+// Script to read an argument and print as lowercase in stdout
+
+use std::env;
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+
+    let user = &args[1];
```

Comment:
```suggestion
    let user = env::args().nth(1).expect("Expected one argument");
```

We can go straight to a particular item in the list of args like this.